scripts/flaky_tests.py

```diff
@@ -0,0 +1,90 @@
+#! /usr/bin/env python3
+
+import argparse
+import json
+import logging
+from collections import defaultdict
+from typing import DefaultDict
+
+import psycopg2
+import psycopg2.extras
+
+# We call the test "flaky" if it changed its status on the same revision in the last N=10 days on main branch.
+FLAKY_TESTS_QUERY = """
+    SELECT
+        DISTINCT ON (parent_suite, suite, name) parent_suite, suite, name
```

Comment:
It seems that you will only have one row per suite. Was it the intention?

Reply:
but what if >1 tests were failed per suite? It was my question.

Reply:
Are you rerunning the whole suite? If so, it's needed to add a comment, that not individual tests will be rerun.

Reply:
Nope, only flaky tests that failed get rerun